src/serac/physics/utilities/functional/CMakeLists.txt

```diff
@@ -5,10 +5,11 @@
 # SPDX-License-Identifier: (BSD-3-Clause)
 
 set(functional_depends serac_physics mfem mpi)
+blt_list_append( TO functional_depends ELEMENTS caliper IF ${SERAC_USE_CALIPER} )
 
-#if(ENABLE_CUDA)
-#    list(APPEND functional_depends cuda)
-#endif()
+if(ENABLE_CUDA)
```

Comment:
Please use blt_list_append here.  

Small side note:  you dont have to use the value of the IF clause, you can send in the variable:

```
blt_list_append( TO functional_depends ELEMENTS caliper IF SERAC_USE_CALIPER )
```